github_pr.py

```diff
@@ -59,7 +59,7 @@ def _print_pr(pr, **args):
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
     else:
-        print "#%d [%s] %10s <- %-30s    %s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title.encode('ascii', errors='ignore'))
+        print "#%d [%s] %10s:%s <- %s:%-30s    %s" % (pr.number, pr.state, pr.base.repo.owner.login, pr.base.ref, pr.head.repo.owner.login, pr.head.ref, pr.title.encode('ascii', errors='ignore'))
```

Comment:
@ferrants looking to get your thoughts on this. The reason for the change is that given a PR number (from an issue comment), I'm using github-pr to get the branch, but if that PR is coming from a fork, I also need the fork. The fork isn't referenced anywhere in issue comments, that's why I need to look it up.
Do you recommend I rev the version of this script since this changes the listing behavior? Also, should I make a similar change to always show the fork for tabular list output, if so do we want a separate column or just combine `fork:branch` into a single column? I'm not using tabular myself so don't need it specifically.